internal/storage/storage.go

```diff
@@ -0,0 +1,54 @@
+package storage
+
+import (
+	"errors"
+
+	"github.com/ilya372317/must-have-metrics/internal/server/entity"
+)
+
+var errAlertNotFound = errors.New("alert not found")
+
+type InMemoryStorage struct {
+	Records map[string]entity.Alert
```

Comment:
```suggestion
	records map[string]entity.Alert
```

Не стоит давать сюда публичный доступ